src/verifact_agents/evidence_hunter.py

```
@@ -10,46 +15,194 @@
+            - Consider the trusted sources in the list: {self.trust_sources}
+            - Assess relevance to the specific claim
+            - Identify the stance (supporting, contradicting, or neutral)
+            - Extract specific passages that directly address the claim
+
+        3. Return a comprehensive set of evidence:
+            - Include supporting, neutral, and contradicting evidence when available
+            - Include multiple evidences on different aspects of the claim
+            - Rank evidence by relevance and credibility (0.0-1.0 scale)
+            - Provide full source information for citation
+            - Include stance classification for each piece of evidence
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)